packages/cli/src/scripts/add.ts

```diff
@@ -4,7 +4,6 @@ import { AddParams } from './interfaces';
 
 export default function add({ contractsData, packageFile }: AddParams): void | never {
   if (contractsData.length === 0) throw new Error('At least one contract name must be provided to add.');
-  contractsData = contractsData.map((data) => typeof(data) === 'string' ? { name: data, alias: data } : data);
```

Comment:
I wouldn't remove this line, as it makes using the `script` from the programmable interface much easier